controllers/controllers.js

```diff
@@ -68,6 +69,22 @@ function postCommentByArticleId(req, res, next) {
     });
 }
 
+function patchArticleById(req, res, next) {
+  const { article_id } = req.params;
+  const { body: newVote } = req;
+
+  if (!newVote.inc_votes) {
+    next({ status: 400, msg: "invalid body" });
```

Comment:
Personally I'd let a psql error handler take care of this, or even ignore it entirely. 

This patch is currently very specific to incrementing votes. While I appreciate that is what the task has asked you to do, you could go on to use this patch to change any part of the article such as editing the body or changing the title.